tools/verdict-census.py

```
@@ -969,6 +976,21 @@ def _rerun_reason(rec, blob):
 
 
+def stale_footer(lines):
+    """The exit line names the cause `stale_check` actually reported.
+
+    ⛔ `stale_check` reports TWO distinct causes — a foreign classifier and a moved population —
+    and one footer saying "the indexed population moved" CONTRADICTS the first, two lines above
+    it. ★ That is CLASS F in this tool's own summary: the discriminating field is in the output
+    and the summarising step throws it away. Extracted so the branch has a control; a footer
+    chosen inline is a branch that regresses in silence.
+    """
+    if any("DIFFERENT classifier" in l for l in lines):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)